README.md

```diff
@@ -1,71 +1,59 @@
-# Getting Started with Create React App
+### BookStore
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+My Bookstore app is a website to show a list of books using a given design and using an API, React, Redux, Javascript and Css to build and style it.
 
-## Available Scripts
+## Built With
 
-In the project directory, you can run:
+# HTML, CSS, JavaScript
 
-### `npm start`
+# React, Redux
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Live Demo
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+not yet...
 
-### `npm test`
+## Getting started
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+To get a local copy up and running follow these simple example steps. (NPM)
 
-### `npm run build`
+1. Open your terminal
+2. Navigate to the directory where you want your project cloned
+3. Clone the project using git clone or just copy the command below:
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+git clone https://github.com/david63011/Bookstore.git
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+4. Open the project with your favorite IDE.
+5. Run npm install to the get the dependencies
+6. Run Webpack and use npm start
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+# Prerequisites
 
-### `npm run eject`
+Having an IDE
+Having Git installed and configured
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+# Setup
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+Clone the repo
+Open the folder with your IDE
+Run npm install
```

Comment:
- [Optional] Consider adding steps to set up `react font awesome` and `react-circular-progress` to your read me so that users can easily use your application.